main/TMP1075.h

```diff
@@ -0,0 +1,14 @@
+#ifndef TMP1075_H_
+#define TMP1075_H_
+
```

Comment:
you need the include <stdbool.h> here because one of your exported function as a bool type